.github/CODEOWNERS

```diff
@@ -54,10 +54,10 @@ util/topgen/        @msfschaffner @tjaychen @Jacob-Levy
 
 
 # DV related common files
-dv/                 @lowRISC/ot-dv-reviewers
-fpv/                @lowRISC/ot-dv-reviewers
-formal/             @lowRISC/ot-dv-reviewers
-# lint/             # TBD
+/hw/**/dv/              @lowRISC/ot-dv-reviewers
+/hw/**/fpv/             @lowRISC/ot-dv-reviewers
+/hw/**/formal           @lowRISC/ot-dv-reviewers
+# /hw/**/lint/                 # TBD
```

Comment:
do we have an `ot-dd-reviewers` for "digital design"? if yes, lint could be assigned to that.

Reply:
Sadly, it does not appear we do. https://github.com/orgs/lowRISC/teams/ot-dd-reviewers is a 404.

Reply:
@moidx @mundaym @tjaychen should we add that list as well?

Reply:
I would put down the usual suspects of RTL reviewers:
@eunchan 
@msfschaffner 
@tjaychen 
@vogelpi 
@GregAC

Reply:
Maybe also @lowRISC/ot-dv-reviewers for one DV rep, since DV code also goes through lint.

Reply:
Following up on this, was the idea to uncomment line 65 and add `ot-dd-reviewers` once it's created? Should i add msfschaffner's list from two comments back in the meantime?